src/Cli/dotnet/commands/dotnet-test/TestApplication.cs

```diff
@@ -24,11 +32,11 @@ public TestApplication(string modulePath, string pipeName, string[] args)
             _args = args;
         }
 
-        public async Task RunAsync()
+        public async Task<int> RunAsync(bool enableHelp)
         {
             if (!ModulePathExists())
             {
-                return;
+                return 1;
```

Comment:
1 is assigned https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-exit-codes